components/Style.qml

```diff
@@ -58,6 +58,7 @@ QtObject {
     property string titleBarCloseSource: blackTheme ? _b_titleBarCloseSource : _w_titleBarCloseSource
     property string titleBarButtonHoverColor: blackTheme ? _b_titleBarButtonHoverColor : _w_titleBarButtonHoverColor
 
+    property string moneroLogoSource: "qrc:///images/monero-vector.svg"
```

Comment:
?

Reply:
updated

wasn’t sure where to add it, I skipped an extra Style.qml entry now